### PR TITLE
Adv-Diff Test

### PR DESCRIPTION
**Pull request by akshaysridhar**
1) Tendency functions
2) Add plot routines and buildkite links
New file:   adv_diff.jl
 Changes to be committed:
	new file:   examples/column/adv_diff.jl

Verification against exact solution.

### CODE REVIEW

**Comment by jakebolewski** [comment]
Looks good to me, just need to reformat the example file: `julia --project=.dev .dev/climaformat.jl examples/column/`

**Comment by jakebolewski** [comment]
bors try

**Comment by jakebolewski** [comment]
bors try-

**Review by jakebolewski — COMMENTED** [review]
Add we need to add an entry for the buildkite CI to run the example:

https://github.com/CliMA/ClimaCore.jl/blob/main/.buildkite/pipeline.yml#L87

File: examples/column/adv_diff.jl [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        top = Operators.Extrapolate(),
+    )
+
+
+    gradc2f = Operators.GradientC2F(bottom = bc_vb,
+                                    top = bc_gt)
+    gradf2c = Operators.GradientF2C()
+
+    return @. dT = gradf2c(ν * gradc2f(T)) - A(V, T)
+end
```
Comment (Zhengyu-Huang) [inline examples/column/adv_diff.jl:93]: This looks correct!

**Review by Zhengyu-Huang — COMMENTED** [review]
Could you have a for loop, which loops the mesh number 2^5, 2^6,...2^8, and compute the error and convergence rate?

**Comment by jakebolewski** [comment]
@Zhengyu-Huang I think that is better as a test (we do this for a bunch of operators).  I'll add that as an additional step.

**Comment by jakebolewski** [comment]
https://user-images.githubusercontent.com/1157798/124839992-21f04a80-df47-11eb-9f13-7c6bca353652.mp4

**Comment by jakebolewski** [comment]
Thanks @akshaysridhar!

**Pull request merged**